### PR TITLE
Add GDT configuration to the kernel

### PR DESCRIPTION
**Pull request by mhouppin**

### CODE REVIEW

**Comment by mhouppin** [comment]
@SystrN7 I'll let you review this before merging

**Comment by tclaudel** [comment]
vu

File: kernel/include/kernel/utils.h
Comment (SystrN7) [inline kernel/include/kernel/utils.h]: Is more relevant to name it kprint_memory ? ?

**Review by SystrN7 — APPROVED** [review]
I don't understand the asm part but it is good for me.

File: kernel/include/kernel/utils.h
Comment (mhouppin) [reply kernel/include/kernel/utils.h]: Fair point, I'll update the function signature.

**Pull request merged**
